Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM golang:onbuild
```

Comment:
`FROM golang:onbuild`
is deprecated and should be replaced by 
`FROM golang`
or 
`FROM golang:alpine`